src/core.js

```diff
@@ -265,14 +265,12 @@ dc.redrawAll = function (group) {
  */
 dc.disableTransitions = false;
 
-dc.transition = function (selections, duration, callback, name) {
-    if (duration <= 0 || duration === undefined || dc.disableTransitions) {
-        return selections;
+dc.transition = function (selections, duration, delay, callback, name) {
+    if (dc.disableTransitions || duration <= 0 || duration === undefined || delay <= 0 || delay === undefined) {
+	return selections;
```

Comment:
Doesn't this enforce a delay? I would think a delay of zero is valid and should still invoke the transition.